corehq/apps/hqadmin/reports.py

```diff
@@ -478,7 +478,9 @@ class UCRDataLoadReport(AdminReport):
     ]
     emailable = False
     exportable = False
-    default_rows = 10
+    disable_pagination = True
+    ajax_pagination = False
+    use_datatables = False
```

Comment:
do we need all these? I thought `use_datatables` was all that was needed?

Reply:
@mkangia 

I don't think they're technically necessary, but I thought of keeping it there to make it obvious to the developer that disabling pagination is the goal, since `user_datatables` doesn't really give me an indication that I'm trying to get rid of pagination. I also thought it's maybe a good idea to be explicit by setting the values to their current state.

Let me know if you think I should remove it or if it's fine to keep it.

Reply:
Okay :+1: 

I am fine with leaving it how you have done it.

I believe the other options make pagination static, i.e all records are loaded on the UI but there is still pagination shown so there is no ajax request to fetch more records but the user still sees pages. I could see that behavior locally.

> I thought of keeping it there to make it obvious to the developer that disabling pagination is the goal

I like that. :+1: